src/_about/faq-devs.md

```diff
@@ -0,0 +1,98 @@
+  accurate names without compromising on quality - and makes it simpler to ask for support in official spaces when
+  you're using a different set of mappings
+* A new Collision Handling ASM backend (CHASM) which acts as the backbone for our mixin and access widener 
+  implementations, allowing those and other bytecode manipulation tools to function without any special handling
+  required in Quilt's build tools or loader, and helping mods to remain compatible with each other
+* A more community-oriented approach that ensures that everyone's voice is heard when raising issues and opinions,
+  regardless of how prolific a developer (or how well-known) they are
+
+{% endadmonition %}
+{% admonition %}
+
+## What does it mean to say that Quilt Mappings doesn't have a clean-room, and why does it matter?
+
+When talking about _**the clean-room**_, we're referring to a specific approach to contributions taken by
+[Fabric's Yarn mappings project](https://github.com/FabricMC/yarn). Yarn takes an approach that attempts to guarantee
+the safety of the project by refusing names that have obviously been inspired by names from other mappings projects,
+such as MCP and Mojang's official mappings. It also discourages contributors from looking at names from other mappings
+projects, and requires discussions in Fabric's official community spaces to be free of non-Yarn names.
+
+Quilt Mappings (shortened to QM below), on the other hand, takes a different approach. As the project uses the 
+[Creative Commons Zero License](https://creativecommons.org/share-your-work/public-domain/cc0/) (similarly to Yarn),
+QM explicitly doesn't claim ownership of the names and instead any names contributed that may be derivatives of other
+mappings projects still have that ownership assigned to them. This, along with including license notices that 
+explicitly show that ownership of some names may lie with other mappings projects (and that provide proper credit) - 
+among other things - allows QM contributors to refer to other mappings projects to figure out what the best name for 
+something should be. It also means that we don't have to disallow discussions involving names from other projects in
+our official community spaces.
+
+It's worth noting, however, that **a name being present in some other mappings project does not mean we'll accept it.**
+All names contributed must stand up on their own, regardless of what Mojang - or any other organisation - provides as
+a name.
+
+{% endadmonition %}
+{% admonition %}
+
+## How hard will it be to port my Fabric mods?
+
+We plan on trying to make this transition as painless as possible - by matching Fabric's API surface, allowing you to
+stick with Yarn mappings if you'd like, and providing support for developers that are porting their mods. It's worth
+noting that - as of this writing - Fabric mods that contain conditional mixins are not supported by Quilt's
```

Comment:
As of writing, this is *not* true--it will work fine until CHASM is implemented.